CHANGELOG.md

```diff
@@ -25,6 +25,14 @@ should change the heading of the (upcoming) version to include a major version b
 
 - Updated the `resolveAllReferences()` function to use object spreading to update properties and items in a schema rather than directly modifying the schema to avoid issues with frozen object, fixing [#3805](https://github.com/rjsf-team/react-jsonschema-form/issues/3805)
 
+## Dev / playground
+
+- update playground vite config to use sources directly, allowing to reload changes in it without additional build step
+- moving from `dts-cli` to use individual dev tools directly, updating package publish config
+  - tsc for generating type definitions and esm modules
+  - esbuild for CJS bundle
+  - rollup for UMD bundle
+
```

Comment:
Leave 5.11.2 and move this into 5.12.0. I needed to release the fix for #3805 to unblock my company

Reply:
done